shared/js/media/video_player.js

```diff
@@ -36,9 +39,11 @@ function VideoPlayer(container) {
   var poster = newelt(container, 'img', 'videoPoster');
   var player = newelt(container, 'video', 'videoPlayer');
   var controls = newelt(container, 'div', 'videoPlayerControls');
-  var playbutton = newelt(controls, 'button', 'videoPlayerPlayButton');
+  var playbutton = newelt(controls, 'button', 'videoPlayerPlayButton',
+                    'playbackPlay');
```

Comment:
As a general thing, I'd prefer wrapped lines like this to be indented so that arguments align with the first argument. That's what my editor does by default, and it looks really weird to me when code does not do that.